data/pc/1.19.3/protocol.json

```diff
@@ -3707,7 +3707,7 @@
             },
             {
               "name": "flags",
-              "type": "i8"
+              "type": "u8"
```

Comment:
Why the data type changes? Does it fix anything broken? If it's applicable to the previous versions you'd have to go back and change all the previous versions' types also.

Reply:
I went through the entire protocol packet by packet and just changed the types as I came across them. As for the ubyte/byte that probably won't make any difference as they usually only require the LSBs, but I figured it makes more sense to use the same types as the Vanilla client.

Reply:
Do previous versions need the same change?

Reply:
To match Vanilla implementation, yes.

Reply:
then can you please change it all ? (or revert for this version)